thewalrus/quantum.py

```diff
@@ -659,6 +659,70 @@ def gen_Qmat_from_graph(A, n_mean):
     return Q
 
 
+def photon_number_var(mu, cov, j, k, hbar=2):
+    r""" Calculate the variance/covariance of the photon number distribution
+    of a Gaussian state.
+

```

Comment:
Could be good to add the details (including the math) that you have in the PR comment!

Reply:
Done! :+1: